lighthouse-core/scripts/update-flow-fixtures.js

```diff
@@ -13,7 +13,10 @@ import {LH_ROOT} from '../../root.js';
 import UserFlow from '../fraggle-rock/user-flow.js';
 
 (async () => {
-  const browser = await puppeteer.launch({headless: false});
+  const browser = await puppeteer.launch({
+    executablePath: process.env.CHROME_PATH,
```

Comment:
I was noticing the FPS was rendered incorrectly on the default puppeteer chrome. We could also update the puppeteer version in this PR. Not sure it started showing up now though.